asv_bench/benchmarks/timeseries.py

```diff
@@ -394,33 +394,42 @@ def time_dup_string_tzoffset_dates(self, cache):
 
 class DatetimeAccessor:
 
-    params = [None, "US/Eastern", "UTC", dateutil.tz.tzutc()]
-    param_names = "tz"
-
-    def setup(self, tz):
+    params = (
+        [None, "US/Eastern", "UTC", dateutil.tz.tzutc()],
+        ["%Y-%m-%d %H:%M:%S.%f%z", "%Y-%m-%d %H:%M:%S%z"],
+        ["T", "S", "NS"],
+    )
+    param_names = ["tz", "fmt", "frequency"]
+
+    def setup(self, tz, fmt, frequency):
         N = 100000
-        self.series = Series(date_range(start="1/1/2000", periods=N, freq="T", tz=tz))
+        self.series = Series(
+            date_range(start="1/1/2000", periods=N, freq=frequency, tz=tz)
+        )
 
-    def time_dt_accessor(self, tz):
+    def time_dt_accessor(self, tz, fmt, frequency):
         self.series.dt
 
-    def time_dt_accessor_normalize(self, tz):
+    def time_dt_accessor_normalize(self, tz, fmt, frequency):
         self.series.dt.normalize()
 
-    def time_dt_accessor_month_name(self, tz):
+    def time_dt_accessor_month_name(self, tz, fmt, frequency):
         self.series.dt.month_name()
 
-    def time_dt_accessor_day_name(self, tz):
+    def time_dt_accessor_day_name(self, tz, fmt, frequency):
         self.series.dt.day_name()
 
-    def time_dt_accessor_time(self, tz):
+    def time_dt_accessor_time(self, tz, fmt, frequency):
         self.series.dt.time
 
-    def time_dt_accessor_date(self, tz):
+    def time_dt_accessor_date(self, tz, fmt, frequency):
         self.series.dt.date
 
-    def time_dt_accessor_year(self, tz):
+    def time_dt_accessor_year(self, tz, fmt, frequency):
```

Comment:
rather than adding these to all of the accessors which don't need this, pls make another class where you just exercise strftime

you can call it DatetimeAccessorStrftime